wikitongues/wikitongues/config/indexing.cfg

```diff
@@ -25,3 +25,7 @@ Sakha : sah
 wikipedia : WikipediaSpider
 fake : fake_spider
 
+[local_config_path]
+windows path : \Wikitongues\Indexing-Project\
+linux path : /Wikitongues/Indexing-Project/
```

Comment:
By default I'd like the config file to live directly in the home directory, outside of the repository. You'd edit this file if you wanted to keep it somewhere else

Reply:
My assumption was that this config was only file in the home directory. So two questions:
1. Are we already saving other data within a Wikitongues/Language-Indexing folder structure in their home directory?
2. If so, then for example, would the mac filepath for the config be something like this?
/home/username/configfile.cfg
instead of this:
/home/username/Wikitongues/Language-Indexing/configfile.cfg

Reply:
1. Nothing else is currently being saved in the home directory
2. Yes, the path on a Mac would be /Users/username/wikitongues-language-indexing.cfg

We wrote a few other command line tools for other tasks that we configure in a similar way, with config files living in the home directory and specifying settings such as API keys or local paths

Reply:
I've updated the script to look directly in the home folder.